app/src/main/java/org/mozilla/fenix/gecko/GeckoProvider.kt

```diff
@@ -58,7 +61,26 @@ object GeckoProvider {
             .contentBlocking(policy.toContentBlockingSetting())
             .debugLogging(Config.channel.isDebug)
             .aboutConfigEnabled(Config.channel.isBeta || Config.channel.isNightlyOrDebug)
-            .build()
+            .configFilePath(
+                (context.filesDir.path + File.separatorChar + GECKO_DEBUG_CONFIG_FILE_NAME).also {
+                    println("Andi: adding \"cookiebanners.listService.testRules\" ... ")
+
+                    File(it).apply {
+                        createNewFile()
+                        printWriter().use {
+                            it.println("prefs:")
+                            it.println("  cookiebanners.listService.testRules: [{\"domains\":[\"localhost\"],\"click\":{\"hide\":\"#cookieBanner\",\"optIn\":\"button#acceptButton\",\"optOut\":\"button#declineButton\",\"presence\":\".cookieBanner\"},\"cookies\":{},\"id\":\"706cca25-cea5-49e8-9179-ff3f55c9c1d3\"}]")
+                        }
+                    }
+
+                    println("Andi: content of $it afterwards:")
+                    println("-------------------------")
+                    println(File(it).bufferedReader().use { it.readText() })
+                    println("-------------------------")
+
+                    println("Andi: Config sent to GV: ${context.filesDir.path + File.separatorChar + GECKO_DEBUG_CONFIG_FILE_NAME}")
+                },
+            ).build()
```

Comment:
@Amejia481 if you don't mind reviewing this bit, thanks

Reply:
This doesn't work @AaronMT , no needs for review.